mkdocs/docs/api.md

```diff
@@ -951,8 +951,10 @@ Using `add_column` you can add a column, without having to worry about the field
 with table.update_schema() as update:
     update.add_column("retries", IntegerType(), "Number of retries to place the bid")
     # In a struct
-    update.add_column("details.confirmed_by", StringType(), "Name of the exchange")
+    update.add_column("details", StructType())
+    update.add_column(("details", "confirmed_by"), StringType(), "Name of the exchange")
```

Comment:
```suggestion

    with table.update_schema() as update:
        update.add_column(("details", "confirmed_by"), StringType(), "Name of the exchange")
```

https://github.com/apache/iceberg-python/pull/1433#issuecomment-2573318869

I opened https://github.com/apache/iceberg-python/issues/1493 to track this issue